README.md

```diff
@@ -248,6 +252,9 @@ of the requests is an array instead of array of arrays.
 * IPROTO constants have been moved to a separate package [go-iproto](https://github.com/tarantool/go-iproto).
 * `PushCode` constant is removed. To check whether the current response is
   a push response, use `IsPush()` method of the response iterator instead.
+* `OkCode` constant is removed.
```

Comment:
Since it's migration guide and not a changelog, I think these two entries should be the one, like "use `ErrorNo` instead of `OkCode` now".